docs/upgrade_1-1_to_1-2.md

```diff
@@ -19,10 +19,21 @@
 
 ## Upgrade steps
 
-### Copy old DPA
+### Backup the DPA configuration
 
 Save your current DataProtectionApplication (DPA) CustomResource config, be sure to remember the values.
 
+For example:
+```
+oc get dpa -n openshift-adp -o yaml > dpa.orig.backup 
+```
+
+### Upgrade the OADP Operator
+
+For general operator upgrade instructions please review the [OpenShift documentation](https://docs.openshift.com/container-platform/4.13/operators/admin/olm-upgrading-operators.html)
+* Change the Subscription for the OADP Operator from `stable-1.1` to `stable-1.2`
+* Allow time for the operator and containers to update and restart
+
 ### Convert your DPA to the new version
 
 If you are using fields that were updated in `spec.configuration.velero.args`, you need to update there new names. Example
```

Comment:
I would cite `Backup the DPA configuration` section here

Reply:
Upgrading the operator is not really tied to the DPA.   I would to prefer to leave it as is.  We'll go through it together and see if it makes sense later.